src/cljs/commiteth/subscriptions.cljs

```diff
@@ -110,16 +112,56 @@
     (get-in db path)))
 
 (reg-sub
- :usage-metrics
-   (fn [db _]
-     (:usage-metrics db)))
+  :usage-metrics
+  (fn [db _]
+    (:usage-metrics db)))
 
 (reg-sub
- :metrics-loading?
-   (fn [db _]
-     (:metrics-loading? db)))
+  :metrics-loading?
+  (fn [db _]
+    (:metrics-loading? db)))
 
 (reg-sub
-    :user-dropdown-open?
+  :user-dropdown-open?
   (fn [db _]
     (:user-dropdown-open? db)))
+
+(reg-sub
+  ::open-bounties-sorting-type
+  (fn [db _]
+    (::db/open-bounties-sorting-type db)))
+
+(reg-sub
+  ::open-bounties-filters
+  (fn [db _]
+    (::db/open-bounties-filters db)))
+
+(reg-sub
+  ::open-bounties-owners
+  :<- [:open-bounties]
+  (fn [open-bounties _]
+    (->> open-bounties
+         (map :repo-owner)
+         set)))
+
+(reg-sub
+  ::open-bounties-currencies
+  :<- [:open-bounties]
+  (fn [open-bounties _]
+    (let [token-ids (->> open-bounties
+                         (map :tokens)
+                         (mapcat keys)
+                         (filter identity)
+                         set)]
+      (into #{"ETH"} token-ids))))
+
+(reg-sub
+  ::filtered-and-sorted-open-bounties
+  :<- [:open-bounties]
+  :<- [::open-bounties-filters]
+  :<- [::open-bounties-sorting-type]
+  (fn [[open-bounties filters sorting-type] _]
+    (cond->> open-bounties
+             filters (ui-model/filter-bounties filters)
+             sorting-type (ui-model/sort-bounties-by-sorting-type sorting-type)
```

Comment:
And also check if `sorting-type` is not the default value: `(not= sorting-type ::ui-model/bounty-sorting-type|most-recent)`.

Reply:
I think that we should consider `:open-bounties` with no order semantic, that is why probably better to apply the sorting-type even if it is the default one.